src/c4/ifpstream.hh

```diff
@@ -0,0 +1,54 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   c4/ifpstream.hh
+ * \author Mathew Cleveland
+ * \brief  Define class ifpstream
+ * \note   Copyright (C) 2018-2020 Triad National Security, LLC.
+ *         All rights reserved. */
+//----------------------------------------------------------------------------//
+
+#ifndef c4_ifpstream_hh
+#define c4_ifpstream_hh
+
+#include <fstream>
+#include <sstream>
+#include <vector>
+
```

Comment:
Remove this  black line unless it is critical for `c4/config.h` to appear after the STL headers.  Normally, we include class headers before STL headers by style and for compile efficiency.